.pre-commit-config.yaml

```diff
@@ -0,0 +1,26 @@
+repos:
+  - repo: https://github.com/python/black
```

Comment:
How about adding the following common hooks to `.pre-commit-config.yaml`

```yml
  - repo: https://github.com/pre-commit/pre-commit-hooks
    rev: v4.3.0
    hooks:
      - id: trailing-whitespace
      - id: end-of-file-fixer
      - id: check-docstring-first
      - id: check-added-large-files
      - id: check-yaml
      - id: check-toml
      - id: requirements-txt-fixer
```

Reply:
added